src/org/thoughtcrime/securesms/RecipientPreferenceActivity.java

```diff
@@ -496,6 +508,48 @@ protected Void doInBackground(Void... params) {
       }
     }
 
+    private class NicknameChangeListener implements Preference.OnPreferenceChangeListener {
+
+      @Override
+      public boolean onPreferenceChange(Preference preference, Object newValue) {
+        NicknameHandler nicknameHandler = new NicknameHandler(getContext());
+
+        try{nicknameHandler.setNickname(recipient, newValue.toString());}
```

Comment:
Same feedback as earlier. Also, what error are you expecting? Could the catch be made more specific?